www/ui-viewport-navigator.html

```diff
@@ -138,6 +138,64 @@
 <!--==================================================================-->
 
 <div class="description">
+    <h3>Navigator Auto Fade</h3>
+    <p>By default, the navigator will auto fade after some timeout when the user stops interacting with the viewport. You can disable this behavior and make it visible persistently.</p>
+
+</div>
+
+
+<div class="demoarea">
+    <div class="demoheading">
+        Viewport with togglable Navigator
+    </div>
+    <button onclick="toggleNav()">Toggle Navigator</button>
+    <div id="contentDiv5" class="openseadragon"></div>
+</div>
+
+<script type="text/javascript">
+    viewer_af = OpenSeadragon({
+        id:            "contentDiv5",
+        prefixUrl:     "/openseadragon/images/",
+        tileSources:   "/example-images/highsmith/highsmith.dzi",
+        showNavigator:  true,
+        navigatorAutoFade: false
+    });
+
+    navShown = true;
```

Comment:
This needs a `var`, as does `viewer_af` above. Likewise their versions in the `pre` below.